.lando.yml

```diff
@@ -69,14 +69,14 @@ services:
     portforward: true
 
 # Proxies
-proxy:
-  appserver:
-    - nnels.local
-  dbadmin:
-    - dbadmin.nnels.local
-  # Not working
-  # indexer:
-  #   - indexer.nnels.local
+# proxy:
```

Comment:
I commented the proxy section out because my site wouldn't start with this in. I don't know why.

Reply:
Not going to delve into this one, the proxy is total minefield.